templates/client-sdk-release/template.yaml

```
@@ -0,0 +1,106 @@
+# Azure DevOps pipeline template to release Client SDK to NPM
+
+parameters:
+
+  - name: 'openapiSpecPath'
+    type: string
+
+  - name: 'projectDir'
+    type: string
+    default: '.'
+
+  - name: 'sdkPackageName'
+    type: string
+    default: ''
+
+  - name: 'dependsOn'
+    type: object
+    default: []
+
+# ------------------
+# Test parameters only
+
+  - name: 'isTestEnvironment'
+    type: boolean
+    default: false
+
+  - name: 'npmRegistry'
+    type: string
+    default: ''
```

Proposal:
we can set the npm registry as default, so we don't have to bother if we are in test env or not
```suggestion
    default: 'https://registry.npmjs.org/'
```

---

Fine for me